### DISCUSSION TITLE
Ability to disable profiles in GHelper

### DISCUSSION THREAD

**Discussion by AbjJeet** [seerge/g-helper#2932] [Q&A]
Hi there,

I have an ASUS G713QR AMD R9 5900HX and RTX3070 from 2021. Some months ago an automatic windows/ASUS BIOS update went wrong and seemingky screwed my BIOS power management. Ever since then, if the laptop goes on any other profile/mode apart from Turbo, it hangs within seconds, and does not even boot into Windows afterward (gets stuck on POST). I have to do a windows recovery to a previous restore point to get it to boot properly again.

Now my issue is that if I accidentally hit the profile switch key on the keyboard, since  I only use the laptop on turbo/custom GHelper profiles, it switches right back to silent/balanced, and the laptop hangs, forcing me to do the above process. Is there a way I can simply disable the silent and balanced profiles? I would still like to switch between turbo and my custom profiles (I have basically made my own silent profile which saves battery by undervolting but also does not hang).

**Comment by seerge** [comment]
@AbjJeet hello, it's quite exotic scenario that you have. Try to do steps from https://github.com/seerge/g-helper/wiki/Troubleshooting, as it may help with your device.

I don't see point in preventing profile selection, but you can force Silent / Balanced modes to also become "Turbo" (as a base) under the hood by adding following lines to the beginning pf `%appdata%\GHelper\config.json`

```
"mode_base_0": 1,
"mode_base_2": 1,
```

This would tell app to use base mode 1 (Turbo) for modes 0 (Balanced) and 2 (Silent)

**Reply by AbjJeet** [reply]
It seems GHelper is overwriting any changes I make to the file. The steps I follow are -
1. Add these lines to the config.json. It seems are two different locations for this file - "%appdata%\GHelper\" and "C:\Program Files (x86)\GHelper" the second one is where I run GHelper.exe from. I added lines to both.
2. Quit Ghelper and end process from taskbar, then start from the "C:\Program Files (x86)\GHelper\Ghelper.exe"

The appdata file seems unchanged, but the file in the "C:\Program Files (x86)\GHelper\" gets overwritten. Did I do anything wrong?

**Reply by seerge** [reply]
@AbjJeet it's only one config location that you need `%appdata%\GHelper`. It should be nothing in Program Files unless you have manually placed it there. If not - you can safely delete that.

For config editing, make sure to quit app **before editing**. 

Check general instructions on how to edit config properly here https://github.com/seerge/g-helper/wiki/Power-user-settings#general-note
